multiversxetl/cli/work.py

```diff
@@ -164,24 +173,25 @@ def load_without_intervals(
 
     Before loading, the data suffers slight transformations (when needed).
     """
-    storage = TasksWithoutIntervalStorage(gcp_project_id)
+    storage = TasksWithoutIntervalStorage(gcp_project_id, group)
 
     do_continuously(
-        lambda: do_any_load_task(workspace, gcp_project_id, storage, worker_id, index_name, schema_folder),
+        lambda: do_any_load_task(gcp_project_id, workspace, group, storage, worker_id, index_name, schema_folder),
         sleep_between_tasks,
         num_threads
     )
 
 
 def do_any_load_task(
-        workspace: str,
         gcp_project_id: str,
+        workspace: str,
+        group: str,
         storage: TasksStorage,
         worker_id: str,
         index_name: Optional[str],
         schema_folder: str
 ):
-    file_storage = FileStorage(Path(workspace))
+    file_storage = FileStorage(Path(workspace) / group)
```

Comment:
what if one uses `  ` (empty string) as the group? Is it a valid directory?

Reply:
Whitespace won't be allowed by the CLI args parser, but I will check this separately, good point (especially for special characters, not necessarily for whitespace).

Empty string will not pass the args parser (since `--group` is required).